src/main/java/org/jabref/gui/preferences/PreviewPrefsTab.java

```diff
@@ -61,25 +57,24 @@
     private final ScrollPane scrollPane = new ScrollPane(layout);
     private final DialogService dialogService;
     private final ExternalFileTypes externalFileTypes;
+    private final TaskExecutor taskExecutor;
+
+    private Task<List<CitationStyle>> citationDiscovery;
 
     public PreviewPrefsTab(DialogService dialogService, ExternalFileTypes externalFileTypes) {
         this.dialogService = dialogService;
         this.externalFileTypes = externalFileTypes;
+        this.taskExecutor = Globals.TASK_EXECUTOR;
         setupLogic();
         setupGui();
     }
 
     private void setupLogic() {
-        chosen.getSelectionModel().selectedItemProperty().addListener(event -> {
-            if (chosen.getSelectionModel().getSelectedIndices().isEmpty()) {
-                btnLeft.setDisable(true);
-                btnDown.setDisable(true);
-                btnUp.setDisable(true);
-            }
-        });
 
-        available.getSelectionModel().selectedItemProperty().addListener(
-                e -> btnRight.setDisable(available.getSelectionModel().getSelectedIndices().isEmpty()));
+        btnLeft.disableProperty().bind(Bindings.isEmpty(chosen.getSelectionModel().getSelectedItems()));
```

Comment:
I think you can define a binding `nothingSelected = Bindings.isEmpty(chosen.getSelectionModel().getSelectedItems())` and then reuse it.